.changeset/hot-knives-arrive.md

```diff
@@ -0,0 +1,5 @@
+---
+"pie-monorepo": minor
+---
+
+[Added] - modal percy token to ci.yml
```

Comment:
Nit: I might be end-of-the-day-misinterpreting but is this changeset still relevant?

Reply:
nope you're correct, this can go :)